core/templates/components/summary-tile/story-summary-tile.component.html

```diff
@@ -250,19 +371,77 @@
       padding: 2vh 3vw 2vh 4vw;
       width: 225px;
     }
-  }
 
-  @media (max-width: 360px) {
-    .story-details {
+    .story-progress-bar {
+      float: right;
+      padding-right: 12px;
+      padding-top: 5px;
+    }
+
+    .chapter-container {
+      background-color: #e6e6e661;
```

Comment:
Is this a hex color code? (We either use rgba format or hex color code format)

Reply:
Done!